mappings/net/minecraft/entity/ai/brain/task/AdmireItemIdleTask.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_4805 net/minecraft/entity/ai/brain/task/AdmireItemIdleTask
```

Comment:
What does "idle" mean? Shouldn't the idle task be the one that doesn't have the `walkSpeed` parameter (which you named `AdmireItemTask`)?

Reply:
Renamed to `WanderWithOffHandItemTask`.